vhost/src/vhost_user/backend_req.rs

```diff
@@ -191,10 +191,16 @@ mod tests {
 
     use super::*;
 
+    fn frontend_backend_pair() -> (Endpoint<VhostUserMsgHeader<BackendReq>>, Backend) {
+        let (p1, p2) = UnixStream::pair().unwrap();
```

Comment:
Instead of using .unwrap() do you want to handle the error here?

Reply:
Well it's utility function only used inside tests, I don't think the error needs to be forwarded here. I think the only situation where this can fail is when maximum file descriptor limit is reached.